### PR TITLE
Add conda-forge package

### LINKED ISSUE #67: Publish the package on conda-forge
I am trying to get xee published on conda-forge. However, the source distribution of the package is not available on PyPI. It would be great to include it. 

image

### PR DESCRIPTION
**Pull request by giswqs**
xee is now available on conda-forge. This PR adds the conda installation instruction and several badges as shown below. 
https://anaconda.org/conda-forge/xee

image

Fixes #67.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Xee: Xarray + Google Earth Engine
 
 [![image](https://img.shields.io/pypi/v/xee.svg)](https://pypi.python.org/pypi/xee)
+[![image](https://static.pepy.tech/badge/xee)](https://pepy.tech/project/xee)
+[![Conda Recipe](https://img.shields.io/badge/recipe-xee-green.svg)](https://github.com/conda-forge/xee-feedstock)
+[![image](https://img.shields.io/conda/vn/conda-forge/xee.svg)](https://anaconda.org/conda-forge/xee)
+[![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/xee.svg)](https://anaconda.org/conda-forge/xee)
```
Comment (alxmrs) [inline README.md:7]: Nit: Can we add all these badges below the subtitle (i.e. line 10)?

**Comment by giswqs** [comment]
Sure. Done. 
image

**Review by alxmrs — APPROVED** [review]
LGTM! :)

**Comment by giswqs** [comment]
Hope to integrate xee with geemap soon. 
https://github.com/gee-community/geemap/issues/1769

**Comment by alxmrs** [comment]
I really want to merge this, I'm currently facing issues with the internal CI system -- it won't let me approve the CL since it thinks I'm a co-author. Nate should be able to resolve this tomorrow morning.

**Comment by giswqs** [comment]
No worries. Take your time.

**Pull request merged**